controllers/complianceeventsapi/server.go

```diff
@@ -284,6 +315,30 @@ func (s *ComplianceAPIServer) Start(ctx context.Context, serverContext *Complian
 	}
 }
 
+func getUserKubeConfig(config *rest.Config, w http.ResponseWriter, r *http.Request) (*rest.Config, error) {
+	userConfig := &rest.Config{
+		Host:    config.Host,
+		APIPath: config.APIPath,
+		TLSClientConfig: rest.TLSClientConfig{
+			CAFile:     config.TLSClientConfig.CAFile,
+			CAData:     config.TLSClientConfig.CAData,
+			ServerName: config.TLSClientConfig.ServerName,
+			// For testing
+			Insecure: config.TLSClientConfig.Insecure,
+		},
+	}
+
+	if r.Header.Get("Authorization") == "" {
```

Comment:
Could you check this on the result of `parseToken`?

This is to return an unauthorized error in the event the user passes `Authorization: Bearer `.